src/qiskit_iqm/fake_backends/chip_sample.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-instance-attributes
+"""
+Abstract representation of an IQM chip sample.
+"""
+from dataclasses import dataclass
+from typing import Any, Union
+
+from .quantum_architectures import IQMQuantumArchitecture
+
+
+@dataclass
+class IQMChipSample:
+    """
+    Provides the specifications of a quantum chip sample.
+
+    Args:
+        quantum_architecture (IQMQuantumArchitecture): an instance of a Quantum Architecture.
+        t1s (dict[int, float]): T1 times for the qubit with the corresponding key.
+        t2s (dict[int, float]): T2 times for the qubit with the corresponding key.
+        one_qubit_gate_depolarization_rates (dict[str, dict[int, float]]): for each one-qubit gate,
+        a depolarization rate is specified for each qubit to match measured gate fidelity.
+        two_qubit_gate_depolarization_rates (dict[str, dict[tuple[int, int], float]]): for each two-qubit gate,
+        a depolarization rate is specified for each coupling to match measured gate fidelity.
```

Comment:
If the description requires several lines, they have to be indented, otherwise Sphinx won't parse the docstring correctly.
```suggestion
        one_qubit_gate_depolarization_rates: for each one-qubit gate,
            a depolarization rate is specified for each qubit to match measured gate fidelity.
        two_qubit_gate_depolarization_rates: for each two-qubit gate,
            a depolarization rate is specified for each coupling to match measured gate fidelity.
```

Reply:
The units of the attributes should also be mentioned, for example by adding `(s)`, `(Hz)` etc.